assets/wire/utils.ts

```diff
@@ -339,3 +339,13 @@ export function getContentTypes(item: IArticle): Set<IContentType> {
 
 export const hasAudio = (item: any) => hasMedia(item, 'audio');
 export const hasVideo = (item: any) => hasMedia(item, 'video');
+
+export function getVersionsLabelText(item: IArticle, plural?: boolean): string {
+    const isTranscript = item.extra?.type === 'transcript';
```

Comment:
Not sure if `extra.type` is typed here, in any case extracting 'transcript' to a constant should be better for future reference